05_customizing-extending-masa/05_m-tag/01_masa-scope-in-dev.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: default
-title: Mura scope in dev
+title: Masa scope in dev
 nav_order: 1
 grand_parent: Customizing/Extending Masa
 parent: M-tag
 last_modified_date: 20-10-2021 13:00
-permalink: /customizing-extending-masa/m-tag/mura-scope-in-dev/
+permalink: /customizing-extending-masa/m-tag/masa-scope-in-dev/
 ---
 
 # Scope in dev
 
-When developing Masa CMS Themes or Plugins the `m` scope is available. This is referred to as the 'Mura Scope'.
-This scope exists next to the familiair `$` scope ('dollar scope').
+When developing Masa CMS Themes or Plugins the `m` scope is available. This is referred to as the 'Masa Scope'.
+The `$` scope ('dollar scope') and `mura` scope are deprecated and should not be used anymore.
```

Comment:
Could you put this in an alert (warning)?
`{: .alert .alert-warning }`

Reply:
Done